functions/go/Makefile

```diff
@@ -11,9 +11,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-SHELL=/bin/bash
+
```

Comment:
If you are trying to avoid using `bash`, why do you still use `bash` in the scripts below?

Reply:
Makefile by default uses `sh`.
I'm trying to avoid setting `SHELL` unless necessary, since in our CD pipeline we may use [docker image](https://hub.docker.com/_/docker) which is alpine-based and it doesn't have `bash` but it has `sh`.
I saw it complained when we set `SHELL=/bin/bash`. Things seems to be working fine after removing this line.

Reply:
If our environment may not have bash, then the shell scripts below shouldn't use bash as well.

Reply:
I have bring this line back. Let's just install bash in alpine.